VAMobile/src/components/CommonErrorComponents/DowntimeError.tsx

```diff
@@ -31,9 +30,12 @@ const DowntimeError: FC<DowntimeErrorProps> = ({ screenID, overrideFeatureName }
     mb: theme.dimensions.contentMarginBottom,
   }
   const { downtimeWindowsByFeature } = useSelector<RootState, ErrorsState>((state) => state.errors)
-  const feature = DowntimeScreenIDToFeature[screenID]
-  const featureName = overrideFeatureName ? overrideFeatureName : downtimeWindowsByFeature[feature]?.featureName
-  const endTime = downtimeWindowsByFeature[feature]?.endTime.toFormat('fff')
+  const feature = ScreenIDToDowntimeFeature[screenID]
+  const downtimeWindow = feature ? downtimeWindowsByFeature[feature] : null
+
+  const featureName = ScreenNameFeatureConstants[screenID]
+  // const featureName = t(`downtime.${screenID}.featureName`)
```

Comment:
Because the feature names are only used for copy, I tried moving them into the translations file but this line caused the linting error: `No overload matches this call.` I assume this means that typescript can't be certain that this will always return a value.